docs/assets/demo.svg

```diff
@@ -14,7 +14,7 @@
             dominant-baseline: text-before-edge;
```

Comment:
Could you please revert this file? I am afraid it will corrupt svg..

Reply:
Updated. Already make `docs/assets/demo.svg` unchanged.